vendor/bat-native-ads/src/bat/ads/internal/ad_targeting/geographic/subdivision/get_subdivision_url_request_builder.cc

```diff
@@ -29,7 +29,7 @@ UrlRequestPtr GetSubdivisionUrlRequestBuilder::Build() {
 ///////////////////////////////////////////////////////////////////////////////
 
 std::string GetSubdivisionUrlRequestBuilder::BuildUrl() const {
-  return base::StringPrintf("%s/v5/getstate", server::GetHost().c_str());
+  return base::StringPrintf("%s/v5/getstate", serve::server::GetHost().c_str());
```

Comment:
this should be /v1/getstate

Reply:
We have always used v5, but checking with amir this worked due to a wildcard. Changing to v1